src/Views/Homepage.vala

```diff
@@ -138,6 +140,13 @@ public class AppCenter.Homepage : Adw.NavigationPage {
 
         updates_badge = new Gtk.Label ("!");
         updates_badge.add_css_class (Granite.STYLE_CLASS_BADGE);
+        fp_client.bind_property (
```

Comment:
Feel free to say "no" if I'm being unreasonable, but I wonder if we should also break out this change to using property bindings here and in AppListUpdateView into a separate commit.

I'm just really nervous about introducing regressions since this part of the codebase is really complicated and historically has been really fragile 😅

Reply:
I opened #2308 for the n updatable and has updatable bindings. There are only two other ones which are one liners and require a bunch more work without the listmodels so I left them for here